common-service-core/src/test/java/com/ca/mfaas/eurekaservice/client/util/UrlUtilsTest.java

```diff
@@ -7,10 +7,13 @@
  *
  * Copyright Contributors to the Zowe Project.
  */
-package com.ca.mfaas.discovery.staticdef;
+package com.ca.mfaas.eurekaservice.client.util;
 
+import com.ca.mfaas.utils.UrlUtils;
 import org.junit.Test;
 
+import java.util.List;
+
 import static org.junit.Assert.*;
 
 public class UrlUtilsTest {
```

Comment:
Needs to cover all test cases for 

- [ ] `getEncodedUrl`
- [ ] `validateUrl`
- [ ] `removeFirstAndLastSlash`
- [ ] `addFirstSlash`
- [ ] `removeLastSlash`